src/tokens/BlessCurseManager.ttslua

```diff
@@ -142,10 +104,9 @@ function initializeState()
     end
 
     -- find tokens in the play area
-    local objs = getObjects()
-    for i, obj in ipairs(objs) do
+    for _, obj in ipairs(getObjects()) do
         local pos = obj.getPosition()
-        if (pos.x > -110 and pos.x < 44 and pos.z > -77 and pos.z < 77) then
+        if pos.x > -50 and pos.x < 50 and pos.z > 8 and pos.z < 50 then
             if obj.getName() == "Bless" then
                 table.insert(tokensTaken.Bless, obj.getGUID())
                 numInPlay.Bless = numInPlay.Bless + 1
```

Comment:
This will miscount if the tokens are stacked, right?

Reply:
It would, but the tokens are not stackable.
![image](https://user-images.githubusercontent.com/97286811/201517294-6933b9ba-d1a3-4d72-9b1d-83f112fa94e7.png)